planning/planning_debug_tools/scripts/perception_replayer/perception_replayer_common.py

```diff
@@ -22,7 +22,7 @@
 from autoware_auto_perception_msgs.msg import DetectedObjects
 from autoware_auto_perception_msgs.msg import PredictedObjects
 from autoware_auto_perception_msgs.msg import TrackedObjects
-from autoware_auto_perception_msgs.msg import TrafficSignalArray
```

Comment:
This change will no longer support past rosbags.
```
$ ros2 run planning_debug_tools perception_replayer.py -b /media/kosuke55/sandisk/rosbag/start_planner_stop_publishing_path/d4d1adc7-4fdf-4613-86b3-0daa2d945b9e_2023-08-25-13-41-44_p0900_5.db3
Stared loading rosbag
[INFO 1694057248.905715364] [rosbag2_storage]: Opened database '/media/kosuke55/sandisk/rosbag/start_planner_stop_publishing_path/d4d1adc7-4fdf-4613-86b3-0daa2d945b9e_2023-08-25-13-41-44_p0900_5.db3' for READ_ONLY. (open() at ./src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp:227)
Ended loading rosbag
Start timer callback
autoware_auto_perception_msgs.msg._traffic_signal_array
Traceback (most recent call last):
  File "/home/kosuke55/pilot-auto/install/planning_debug_tools/lib/planning_debug_tools/perception_replayer.py", line 189, in <module>
    rclpy.spin_once(node, timeout_sec=0.01)
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/__init__.py", line 202, in spin_once
    executor.spin_once(timeout_sec=timeout_sec)
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 713, in spin_once
    raise handler.exception()
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/task.py", line 239, in __call__
    self._handler.send(None)
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 418, in handler
    await call_coroutine(entity, arg)
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 332, in _execute_timer
    await await_or_execute(tmr.callback)
  File "/opt/ros/humble/local/lib/python3.10/dist-packages/rclpy/executors.py", line 107, in await_or_execute
    return callback(*args)
  File "/home/kosuke55/pilot-auto/install/planning_debug_tools/lib/planning_debug_tools/perception_replayer.py", line 102, in on_timer
    traffic_signals_msg.stamp = timestamp
AttributeError: 'TrafficSignalArray' object has no attribute 'stamp'
[ros2run]: Process exited with failure 1
```

Reply:
support old auto TrafficSignalArray 

https://github.com/autowarefoundation/autoware.universe/pull/4910